src/Approximations/template_directions.jl

```diff
@@ -330,3 +337,139 @@ end
 
 end # @eval
 end # if
+
+# ==================================================
+# Spherical directions
+# ==================================================
+
+# returns `true` if and only if there is a vector in the list of vectors S
+# such that it is approximately d
+function _isapprox_included(d::Vector{N}, S::Vector{Vector{N}}) where {N}
+    result = false
+    for si in S
+        if isapproxzero(norm(d-si))
+            result = true
+            break
```

Comment:
Slightly shorter: `return true` here and `return `false` below (saves the `result` variable).